firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -73,8 +73,8 @@ class SearchViewController: SiteTableViewController,
     private let viewModel: SearchViewModel
     private let model: SearchEngines
     private var suggestClient: SearchSuggestClient?
-    private var remoteClientTabs = [ClientTabsSearchWrapper]()
-    private var filteredRemoteClientTabs = [ClientTabsSearchWrapper]()
```

Comment:
nit: seems like we're exposing these variable only for tests, is there a way to test without making these public?

Reply:
I wonder if a good middle ground would be to make these `internal`, instead of `private` or public? (Or keep these `private` and add `internal` getter functions, if we wanted to avoid exposing mutable `var`s?)

We already use the [`@testable` attribute](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#testable) in [`SearchViewControllerTests`](https://github.com/mozilla-mobile/firefox-ios/blob/5400856aa4c3b372bef473be038ca6084a39ab94/firefox-ios/firefox-ios-tests/Tests/ClientTests/SearchViewControllerTests.swift#L10), so that'll let those tests access `internal` members.

Reply:
these should be `internal` by default if we don't explicitly state them

Reply:
this is not a blocker for this ticket in my opinion, we can create a separate ticket, so I approved based on everything else looking good!